_config/dirs.txt

```diff
@@ -1,2 +0,0 @@
-/rfanfs/pnl-zorro/
-tb571@eris1n3.research.partners.org:/data/pnl
```

Comment:
@colinjennings , this is still incorrect. Bring back our original `_config/dirs.txt`. I see that you added it to `.gitignore` already. So its changes will get out of your way in future.

Reply:
Reverted to original state and used `git update-index --assume-unchanged _config/dirs.txt` to make sure the contents are not tracked